test-infra/image-builder/roles/kubevirt-images/tasks/main.yml

```diff
@@ -13,6 +13,14 @@
   with_dict:
     - "{{ images }}"
 
+- name: Unxz compressed images
+  command: unxz --force {{ images_dir }}/{{ item.value.filename }}
+  with_dict:
+    - "{{ images }}"
+  when:
+    - item.value.filename.find('.qcow2.xz') != -1
```

Comment:
I think `.endswith()` would be more explicit?

Reply:
Yes, you're right.